Gemfile

```diff
@@ -40,8 +49,15 @@ end
 group :development do
   # Access an IRB console on exception pages or by using <%= console %> in views
   gem 'web-console', '~> 2.0'
+  gem 'better_errors'
+  gem 'pry-rails'
+  gem 'binding_of_caller'
+  gem 'rails-erd'
 
```

Comment:
I used the Bullet gem in the Betsy project and it helped me learn about what was going on in the back end in terms of what queries were happening to load each page. Totally not required, but I found it really interesting so I thought I'd share.
http://railscasts.com/episodes/372-bullet